packages/scandipwa/src/component/ProductPrice/ProductPrice.component.js

```diff
@@ -367,14 +367,17 @@ export class ProductPrice extends PureComponent {
 
     renderDefaultPrice(defaultLabel = null) {
         const {
+            originalPrice: { minFinalPriceExclTax = {} } = {},
             price: { finalPrice = {}, finalPriceExclTax = {} } = {},
             label
         } = this.props;
 
+        const priceExclTax = finalPriceExclTax.value === 0 ? minFinalPriceExclTax : finalPriceExclTax;
```

Comment:
You shouldn't just add a workaround of switching one field to another, if for some reason the field that works fine for other products/product types doesn't return a correct value. 

What you should take into account when developing a proper fix:
There is a `getPrice` method that adjusts price calculations for selected product or product variant: https://github.com/scandipwa/scandipwa/blob/master/packages/scandipwa/src/component/Product/Product.container.js#L245
When you deal with a configurable product, prices for product variant are calculated instead of parent product.
Potentially there can be an error if that logic for getting price from product variant is applied, when you add a configurable product to wishlist, but there is no selected variant, as you didn't select configuration before adding to wishlist.

You can see that before applying `getPrice` calculations, price data excl. tax was correct:

![Monosnap 2022-01-03 13-42-31](https://user-images.githubusercontent.com/79456428/147937556-9de68a64-691f-44ec-9484-e27aaf6c656d.png)

But in `output` returned by `getPrice`, we see `0` for price excl. tax:
![Monosnap 2022-01-03 13-43-09](https://user-images.githubusercontent.com/79456428/147937696-23839daf-8bb6-4974-b970-9041306e89a1.png)

=> This method doesn't work correctly with configurable products without selected configurations.